src/Shotr.Ui/Forms/MainForm.cs

```diff
@@ -920,6 +920,7 @@ private void resetSettingsButton_Click(object sender, EventArgs e)
             {
                 SettingsService.Reset();
                 Application.Restart();
+                Environment.Exit(0);
```

Comment:
This did the trick. 

Note: if for some reason we get complaints about it force restarting then we can just reuse the HotKey set functions. I'm fine with this implementation for now.